DependencyInjection/CmfCreateExtension.php

```diff
@@ -86,20 +86,25 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadSecurity($config['security'], $loader, $container);
 
-        $hasMapper = false;
         if ($config['persistence']['phpcr']['enabled']) {
             $this->loadPhpcr($config['persistence']['phpcr'], $loader, $container);
-            $hasMapper = true;
         } else {
             // TODO: we should leverage the mediabundle here and not depend on phpcr
             $container->setParameter($this->getAlias() . '.image_enabled', false);
         }
+        if ($config['persistence']['orm']['enabled']) {
+            $this->loadOrm($config['persistence']['orm'], $loader, $container);
+        }
+        $hasMapper = false;
         if (isset($config['object_mapper_service_id'])) {
```

Comment:
i suggest we simply make chain_mapper the default value for this configuration and get rid of the condition and also the whole hasMapper check.